### ISSUE TITLE
톰의 외형은 어떤 것이 맞나요?

### ISSUE THREAD

**Issue by daew0n** [daew0n/cau-osp2020-novel#1157]
톰의 색상과 무늬는 아래 조합 중 어떤 것이 맞나요?

1. 회색/검정 줄무늬
2. 갈색/검정 줄무늬
3. 기타

**Comment by eunyoungyi** [comment]
발단 초반 부분에 검은색과 회색의 털로 이루어진 줄무늬 고양이로 묘사되어 있습니다!!

**Comment by gh-BumsooKim** [comment]
제가 모습을 구체화하기 위해서 발단 도입부에 벵갈 고양이로 설정해주었는데, 이후에 검은색과 회색 털의 줄무늬를 가진다는 내용이 추가된 듯 합니다.

**Comment by daew0n** [comment]
고양이를 톰 만화 캐릭터로 묘사하는 경우 줄무늬 묘사와 어긋나게 되네요

**Comment by gh-BumsooKim** [comment]
네 맞습니다. 이후 삽화를 위해서 톰의 묘종과 함께 무늬/색, 체형을 확고하게 할 필요가 있겠네요. 이 외에 다른 등장 인물들도 외관을 정하는게 좋을거 같습니다.
